search/search_quality/assessment_tool/sample_view.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "search/result.hpp"
+
+#include <QtWidgets/QWidget>
+
+class LanguagesList;
+class QLineEdit;
+class QListWidget;
+
+namespace search
+{
+struct Sample;
+}
+
+class SampleView : public QWidget
```

Comment:
It's possible but do we really need it? I mean, when you develop a small application, do you worry about namespaces? This is not a library code.